Makefile

```diff
@@ -8,16 +8,10 @@ FS_MODULES=/usr/lib/freeswitch/mod
 SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
-CC=gcc 
 CFLAGS=-fPIC -O3 -fomit-frame-pointer -fno-exceptions -Wall -std=c99 -pedantic
-ifeq (${PROC},x86_64)
```

Comment:
It breaks compilation on 64BIT arm architectures.

If people really need that they can specify it in the make commandline.
make CFLAGS=-m32